ktor-client/ktor-client-core/common/test/HttpStatementTest.kt

```diff
@@ -0,0 +1,20 @@
+import io.ktor.client.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import kotlin.test.*
+
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+class HttpStatementTest {
+
+    @Test
+    fun testStatementToString() {
```

Comment:
The copied encode function is not covered by tests so could be potentially broken in the future. Shouldn't we copy tests as well?

Reply:
Sorry it's still draft. Forgot to switch the mode